Shell/Options.cpp

```diff
@@ -92,6 +92,11 @@ void Options::init()
   _instructionLimit = UnsignedOptionValue("instruction_limit","i",0);
   _instructionLimit.description="Limit the number (in millions) of executed instructions (excluding the kernel ones).";
   _lookup.insert(&_instructionLimit);
+
+  _parsingDoesNotCount = BoolOptionValue("parsing_does_not_count","",false);
```

Comment:
Cute, I could use this!

Reply:
Just note it only works with instruction limit at the moment.